provider/test_utils.go

```diff
@@ -75,3 +76,14 @@ func TestCheckResourceAttrNot(name, key, value string) resource.TestCheckFunc {
 		return nil
 	}
 }
+
+func keycloakVersionIsHigherOrEqualThan(keycloakMajorVersion int) (bool, error) {
```

Comment:
nitpick: could you rename this to `keycloakVersionIsGreaterThanOrEqualTo`?

Reply:
done